web-app/js/projects.js

```diff
@@ -773,13 +778,58 @@ function CreateEditProjectViewModel(project, isUserEditor, userOrganisations, or
         var here = document.location.href;
         document.location.href = config.organisationCreateUrl+'?returnTo='+here+'&returning=true';
     };
+
     self.organisationSearch.selection.subscribe(function(newSelection) {
         if (newSelection) {
             self.organisationId(newSelection.organisationId);
         }
     });
 
-    self.ignore = self.ignore.concat(['organisationSearch']);
+    self.associatedOrganisationSearch.addSelectedOrganisation = function() {
+        var org = { id: self.associatedOrgs().length };
+
+        if (self.transients.associatedOrgNotInList()) {
+            org.name = self.associatedOrganisationSearch.term();
+            org.url = self.transients.associatedOrgUrl() || null;
+            org.logo = self.transients.associatedOrgLogoUrl() || null;
+        } else {
+            var logoDocument = ko.utils.arrayFirst(self.associatedOrganisationSearch.selection().documents, function(document) {
+                return document.role === "logo"
+            });
+
+            org.organisationId = self.associatedOrganisationSearch.selection().organisationId || "";
+            org.name = self.associatedOrganisationSearch.selection().name;
+            org.url = self.associatedOrganisationSearch.selection().url || null;
+            org.logo = logoDocument && logoDocument.thumbnailUrl ? logoDocument.thumbnailUrl : null;
+        }
+
+        self.associatedOrgs.push(org);
+        self.associatedOrganisationSearch.clearSelection();
+        self.transients.associatedOrgLogoUrl(false);
+        self.transients.associatedOrgUrl(null);
+        self.transients.associatedOrgLogoUrl(null);
+    };
+
+    self.removeAssociatedOrganisation = function(data, event) {
+        // the id is stored in the data-value attribute of the button to work around the Knockout limitation of not being
+        // able to pass parameters to click event handlers.
+        var target = null;
```

Comment:
you can use first function parameter - data - to do remove from list. i.e. self.associatedOrgs.remove(data)